### PR TITLE
Overhaul site planning document.

### PR DESCRIPTION
**Pull request by bbockelm**
Focus currently is deleting absolutely irrelevant / incorrect content and including as many tidbits about OSG as possible.

The result is not pretty - it looks a bit like a wall of words. Diagrams need to be added back, as well as some tables to help illustrate the decision tree.

Have to start somewhere!  There are a lot of things I don't like about this rewrite, but there's less than what we started with.  We at least don't have to put up a warning to readers to not read the document.

### CODE REVIEW

File: docs/site-planning.md [hunk 1]
```diff
@@ -1,235 +1,121 @@
...
+independently-managed computing and storage resources. These individual services will be accessed by virtual
+organizations (VOs), which will delegate the resources to scientists, researchers, and students.
 
-A **Compute Element** allows grid users to run jobs on your site. It is software that provides following services when run on your **gatekeeper**: The standard installation is based on HTCondor-CE with RSV for monitoring.
+_Sharing_ is a fundamental principle for the OSG: your site is encouraged to support as many OSG-registered VOs as
+local conditions allow.  _Autonomy_ is another principle: you are not required to support any you do not want.
 
-You must determine your security policy with regard to Unix ID management on the cluster. You may choose group accounts and/or dynamic accounts for all users.
+As the administrator responsible for deployment of the OSG software stack, your task is to make your existing computing
+and storage resources available to and reliable for your support VOs. Fundamentally, there are three components:
```
Comment (brianhlin) [inline docs/site-planning.md]: Did you mean '...reliable for your supported VOs'?

File: docs/site-planning.md [hunk 2]
```diff
@@ -1,235 +1,121 @@
...
+Worker Node Client
+------------------
+
+The Worker Node Client is software installed on each worker node to provide a minimal runtime environment for pilot
+jobs.  We strive to keep this as lightweight as possible; it includes basic grid utilities, not scientific libraries.
+
+There is a wide range of worker node management practices; accordingly, we provide the worker node client in three
+forms:
+
+- [*Classical RPM packaging*](worker-node/install-wn.md): Grid utilities are deployed at the system level using `yum`.
```
Comment (brianhlin) [inline docs/site-planning.md]: "Classical" seems unnecessary here

File: docs/site-planning.md [hunk 3]
```diff
@@ -1,235 +1,121 @@
...
+resources regardless, but installing CVMFS is a primary mechanism to attract more jobs.
+
+Data Service
+------
+
+There are two types of data services on the OSG:
+
+- A *cache* pulls in data on read-demand, keeping it locally for further use.  We provide support for the
+  [Squid HTTP](data/frontier-squid) cache for smaller working set sizes and software called [Xrootd](http://xrootd.org)
+  for otherwise.  Almost every site on the OSG installs a Squid cache.
```
Comment (brianhlin) [inline docs/site-planning.md]: "for otherwise"?

**Comment by bbockelm** [comment]
@brianhlin - I'm merging this word wall over the existing "diagrams of wrongness" page.

Let's touch base with FKW and see if we can't get a few of the diagrams of site deploy options he uses in his various presentations and get them on this page.

**Pull request merged**